docs/docs/developers/guides/smart_contracts/writing_contracts/storage/storage_slots.md

```diff
@@ -31,8 +31,8 @@ sequenceDiagram
     BalanceSet->>Set: insert(note)
     Set->>LifeCycle: create_note(derived_slot, note)
     LifeCycle->>LifeCycle: note.header = NoteHeader { contract_address, <br> storage_slot: derived_slot, nonce: 0, note_hash_counter }
-    Utils->>UintNote: note_hiding_point = note.compute_note_hiding_point()
-    UintNote->>Utils: note_hash = note_hiding_point.x
+    Utils->>UintNote: partial_note_commitment = note.compute_partial_note_commitment()
```

Comment:
I don't see a method like this on `note`. It might be better to specify that `compute_partial_commitment` is a method on PartialUintNote (https://github.com/AztecProtocol/aztec-packages/blob/27eed71c797f86e2190e14bb95290d3c73131564/noir-projects/aztec-nr/uint-note/src/uint_note.nr#L55)